Dockerfile

```diff
@@ -36,4 +36,4 @@ RUN python3 -m venv .venv && \
 
 ENV PYTHONPATH="/openvdb/build/openvdb/openvdb/python"
 
-ENTRYPOINT [ "sleep", "infinity" ]
+ENTRYPOINT [ "sleep", "infinity" ]
```

Comment:
Not sure why this was taken out in the first place. Will be overwritten by Zach's incoming dockerfile regardless

Reply:
Looks like it was taken out then put back in?